NewHaven/test/ResourcesDriver.cpp

```diff
@@ -3,15 +3,39 @@
 
 int main()
 {
-    HarvestTile tile;
-    resourceTypes* content{ tile.getTileContent() };
-    std::uint_least8_t* position{ tile.getPosition() };
+    HarvestDeck deck;
+    HarvestTile* tile;
+    resourceTypes* content;
+    std::uint_least8_t deckSize;
+    std::uint_least8_t* position;
 
+    for (std::uint_fast8_t i = 0; i < 60; i++)
```

Comment:
Okay, so I ran the code and everything compiled fine. However, the output is strange. I don't know if this is done on purpose or not but each time a card is drawn the new card is identical to the previous? Shouldn't each card in the deck be randomized? Duplicates are bound to happen but since we are working with 60 cards not all 60 of them should be the same.